### DISCUSSION TITLE
Landscape mode navigation bar width

### DISCUSSION THREAD

**Discussion by Markus40** [organicmaps/organicmaps#2544] [Q&A]
Is it intentional that the bottom navigation bar is narrow and does not take up the entire width of the screen when the phone is in landscape mode? It changed in the last update.

**Comment by biodranik** [comment]
Can you please share a screenshot?

**Reply by Markus40** [reply]
![Screenshot](https://user-images.githubusercontent.com/83376570/167960294-2aeb50c8-5595-4b36-a5f5-61ce109afdc1.jpg)

**Reply by biodranik** [reply]
As I remember, we discussed that change. Don't you like it? Why?

**Reply by Markus40** [reply]
I have only been following the project for a short time and did not catch the discussion. The change wasn't listed individually in the patch notes and I didn't find the change in the closed issues, so I wondered if it was a bug.

I'm relatively neutral towards it, it just looks a bit weird.

**Reply by biodranik** [reply]
Please check #2258
We just forgot to mention it in whats new )

**Reply by Markus40** [reply]
Thanks

**Reply by Donkey-Doug** [reply]
I think the change is good, but the top bar being full width is weird. Can we include the street name in the arrow and distance square like this:

![PS0AAAAASUVORK5CYII=](https://user-images.githubusercontent.com/50382414/168402025-0d5644d2-896d-4c13-a5e1-73f3d5f31c44.png)

(sorry for weird proportions)

**Reply by biodranik** [reply]
Yes, I think there were already some issues about it.
Also added to #1104

**Comment by Donkey-Doug** [comment]
I suggest making it vertical and on the side.

**Reply by biodranik** [reply]
That's an interesting idea. What to do with buttons? Maybe integrate them into (other?) toolbar? Any ideas and mockups are welcome!

**Reply by Donkey-Doug** [reply]
Something like this:

![carplay-ios-14-verbeteringen](https://user-images.githubusercontent.com/50382414/168400649-b7e507d7-0970-4671-83e1-b5c47eb94866.jpg)

Have all buttons in a vertical bar that takes up less space.

**Reply by biodranik** [reply]
On some [android] cars the left panel is hidden by the too-high border of the device if the driver sits on the left. Hopefully, it's only on a few cars.
